wiki/Sitemap/en.md

```diff
@@ -0,0 +1,270 @@
+# Sitemap
+
```

Comment:
Add something like "Complete list of osu!wiki articles. Updated every month."

Reply:
I don't think it would be every month, once the bot is live it would be updated every 15 minutes

Reply:
sitemap is and should already be self-explanatory... a leading paragraph for this isn't needed imo